practice.rb

```diff
@@ -0,0 +1,66 @@
+def print_array(a)
+  print "["
+  l = a.length
+  a.each do |item|
+    print "#{item}"
+    if item != a[l-1]
+      print ", "
+    end
+  end
+  print "]"
+end
+
+print_array([4,5,6,4,2,123,42])
+
+def stretch(list)
+  print "["
+  l = (list.length) * 2
```

Comment:
ugh. this is super nit picky, but I greatly dislike the use of lowercase L as a variable name because my eyes trick me sometimes and I can't tell if it's a 1 or an l.